### PR TITLE
Chainsaw now consumes welding fuel

### PR DESCRIPTION
**Pull request by MilenVolf**
## About the PR
- Chainsaw is now toggleable and consumes welding fuel to work.
- Some sprite, prototype and audio improvements.
- Chainsaw works about 2 minutes with 300u of Welding Fuel.

## Why / Balance
I think that it makes more sense.
Maybe later will make it more accessible or maybe craftable but less deadly, 'cause for now, only cargo can get it with pretty low chance in lotlery.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/space-wizards/space-station-14/assets/63782763/e07fefcd-db2a-465e-9b33-32a33c72f986

**Changelog**

:cl:
- tweak: Chainsaw now requires welding fuel to work.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/chainsaw.yml [hunk 1]
```diff
@@ -44,6 +75,9 @@
         reagents:
         - ReagentId: WeldingFuel
           Quantity: 300
-        maxVol: 300
+  - type: Welder
+    fuelConsumption: 3
+    fuelLitCost: 2
+    tankSafe: true
   - type: UseDelay
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/chainsaw.yml:82]: Please don't use weldercomponent for this, split it out into a reagent provider.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by MilenVolf** [comment]
Not even sure about cl. Almost nobody even know that chainsaw exist.

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/chainsaw.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/chainsaw.yml:82]: See also https://github.com/space-wizards/space-station-14/issues/18689

Comment (MilenVolf) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/chainsaw.yml:82]: > See also #18689

Thanks. Then I should deep dive a little into ToggleSystem. Will try to make required component for cases like this.

**Comment by IProduceWidgets** [comment]
300u seems like a lot. welding carts only have 1500 iirc, and this is 3x larger than a large beaker.

**Comment by metalgearsloth** [comment]
Re-open if you come back to it.

**Pull request closed without merging**